pkg/providers/instance/instance.go

```diff
@@ -166,6 +167,11 @@ func (p *Provider) Delete(ctx context.Context, id string) error {
 		}
 		return fmt.Errorf("terminating instance, %w", err)
 	}
+	for _, instance := range response.TerminatingInstances {
```

Comment:
What are the possible states for the response here? Do we sometimes get a length of 1 response and sometimes get a length of 0? Or is always 1?

Reply:
I think we will always get 1 here.